dcrjson/dcrwalletextcmds.go

```diff
@@ -532,6 +534,30 @@ func NewWalletInfoCmd() *WalletInfoCmd {
 	return &WalletInfoCmd{}
 }
 
+type OutputDestination struct {
+	Address       string
+	Script        []byte
+	ScriptVersion uint32
+}
+
+// SweepAccountCmd defines the sweep account JSON-RPC command.
+type SweepAccountCmd struct {
+	SourceAccount         uint32
+	RequiredConfirmations uint32
+	FeePerKb              dcrutil.Amount
+	ChangeDestination     *OutputDestination
+}
+
+// NewSweepAccountCmd returns a new instance which can be used to issue a JSON-RPC SweepAccountCmd command.
+func NewSweepAccountCmd(sourceAccount uint32, requiredConfs uint32, feePerKb *dcrutil.Amount, changeDestination *OutputDestination) *SweepAccountCmd {
```

Comment:
after reordering the request parameters in the struct, these should be reordered as well.